### PR TITLE
Lidar support

### PR DESCRIPTION
**Pull request by MoffKalast**
Adds the stock hokuyo lidar support (and rviz laserscan to config) and disables it by default.

The enabling param is `lidar_installed` in `magni_description/urdf/magni.urdf.xacro`.

Also fixes the camera topics so it now mirrors the real Pi setup.

### CODE REVIEW

**Review by JanezCim — COMMENTED** [review]
Compiles and starts easily on Melodic. It does produces a strange lidar output where it detects stuff very close to the lidar. Picture is sent to Vid, because github doesnt allow uploading pngs

**Comment by MoffKalast** [comment]
> because github doesnt allow uploading pngs



Are you sure about that? :D

Anyhow yeah this should be further investigated, though I'm not sure if it actually causes any issues with mapping.

**Comment by JanezCim** [comment]
> Are you sure about that? :D

Ok, then i have to solve a problem at my end

> Anyhow yeah this should be further investigated, though I'm not sure if it actually causes any issues with mapping.

I agree, but i dont think its smart to merge before its solved

**Comment by MoffKalast** [comment]
Okay, this seems to be solved with an increased lidar min distance of 0.2m which doesn't seem to affect detection too much. I've also added the lidar param to all related launch files in line with the sonar convention.

File: magni_bringup/launch/base.launch [hunk 1]
```diff
@@ -5,9 +5,18 @@
...
 -->
 <launch>
+    <!-- 
+        This parameter is used to specify what level of capabilites
+        the robot should have after this launch file.
 
-    <node name="move_basic" pkg="move_basic" type="move_basic" output="screen">
+        Possible values are 'core', 'teleop', and 'navigation'
 
-	<remap from="/scan" to="/fuckoffff"/>
```
Comment (JanezCim) [inline magni_bringup/launch/base.launch]: classy xD

**Review by JanezCim — APPROVED** [review]
Now works ok on melodic :)

File: magni_bringup/launch/base.launch (hunk 1 above)
Comment (MoffKalast) [reply magni_bringup/launch/base.launch]: I was hoping nobody would notice that I accidentally committed and then removed that :P 

Anyhow that's the result of me losing my mind at move_basic detecting all walls and not moving at all for the umpteenth time that evening with a deadline of -1 hours left.

**Pull request merged**